### PR TITLE
On add-on first run, open Test Pilot if not already open (closes #1374).

### LINKED ISSUE #1374: On add-on first run, open Test Pilot
Breaking out from #1360:

On add-on first run, if there are no open test Test Pilot tabs, one should be opened. This is primarily intended to cover the restart-required use case, but may be more broadly useful.

### PR DESCRIPTION
**Pull request by chuckharmston**
~~Not ready for review.~~

### CODE REVIEW

**Comment by chuckharmston** [comment]
This is ready for review now.

File: addon/index.js [hunk 1]
```diff
@@ -358,6 +359,7 @@ exports.main = function(options) {
   ToolbarButton.init(settings);
   ExperimentNotifications.init();
   SharePrompt.init(settings);
+  FirstRun.setup(options.reason, settings);
```
Comment (lmorchard) [inline addon/index.js:362]: I can't seem to find the issue now, but I remember you opening one about making this general module pattern cleaner and more like Django. :+1: to this and that in general

File: addon/test/test-first-run.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
+/*
```
Comment (lmorchard) [inline addon/test/test-first-run.js:1]: :+1: Yay tests, and these pass for me locally.

**Review by lmorchard — APPROVED** [review]
Works for me, looks clean & good

**Pull request merged**